### PR TITLE
index adapted to screen size

### PR DESCRIPTION
**Pull request by casantadu**
Not many changes here

### CODE REVIEW

**Review by ymik0410 — COMMENTED** [review]
Please remove buttons

**Comment by casantadu** [comment]
Done

File: app/assets/stylesheets/components/_index.scss [hunk 1]
```diff
@@ -3,3 +3,4 @@
 @import "avatar";
 @import "navbar";
 @import "cards";
+@import "buttons";
```
Comment (ymik0410) [inline app/assets/stylesheets/components/_index.scss]: please remove it as well

**Comment by casantadu** [comment]
now done

**Review by ymik0410 — COMMENTED** [review]
Santa, super! just remove that line, please

**Pull request merged**